src/Components/NavBar/Menus/NavBarUserMenuAvatar.tsx

```diff
@@ -0,0 +1,50 @@
+import { Avatar, Skeleton } from "@artsy/palette"
+import { FC } from "react"
+import { graphql, useLazyLoadQuery } from "react-relay"
+import { NavBarUserMenuAvatarQuery } from "__generated__/NavBarUserMenuAvatarQuery.graphql"
+
+interface NavBarUserMenuAvatarProps {}
+
+export const NavBarUserMenuAvatar: FC<NavBarUserMenuAvatarProps> = () => {
+  const { me } = useLazyLoadQuery<NavBarUserMenuAvatarQuery>(
```

Comment:
Why use this over `useClientQuery`? (Maybe I already asked this elsewhere, forgive)

Reply:
Caching doesn't work correctly with `useClientQuery`

Reply:
oh that's good to know -- is that a hypothesis, or just going off of the fetchQuery docs?

Reply:
That, I don't know. I didn't notice anything in the docs. Seems like it might be a bug; but also in general I don't mind using `useLazyLoadQuery` so it seems like we should move over anyway.